### PR TITLE
ames: properly migrate old %snub tasks

### PR DESCRIPTION
**Pull request by pkova**
The `%snub` task was introduced in `urbit-os-v2.131`: https://github.com/urbit/urbit/blob/urbit-os-v2.131/pkg/arvo/sys/lull.hoon#L379

This corresponds to `%ames` state `%9`: https://github.com/urbit/urbit/blob/urbit-os-v2.131/pkg/arvo/sys/vane/ames.hoon#L863

It was later modified in in `urbit-os-v2.136`: https://github.com/urbit/urbit/blob/urbit-os-v2.136/pkg/arvo/sys/lull.hoon#L383

Corresponding `%ames` state is `%12`: https://github.com/urbit/urbit/blob/urbit-os-v2.136/pkg/arvo/sys/vane/ames.hoon#L884

The `+load` migration was correctly written from `%ames` state `%11` to `%12`, but given that the `queued-event` type is dependent on the type of `task:ames` in lull, versions `%9` and `%10` will miserably fail to migrate, printing out the type of ames state for hours.

TLDR it's currently impossible to update  your `urbit-os` from a few versions back, unless you know how to manually do it one version at a time. This PR fixes the problem.

I've tested this  on multiple livenet ships that experienced this problem. I've also tested it with all relevant `urbit-os` combinations using fakezods booted from old pills.

### CODE REVIEW

**Review by belisarius222 — APPROVED** [review]

**Comment by vvisigoth** [comment]
This is still broken on my ship.

Background:

I was encountering this error whenever my ship tried to update to 414. At @matthew-levan suggestion I downgraded to 2.0 and the problem went away.

Seeing this hotfix was shipped, I switched to 2.1 and ran `|bump` the upgrade eventually hits a `clay: wick` error and then print out what appears to be the entire ames state, hanging for over an hour.

Please tell me how I can upgrade to 414.

**Comment by joemfb** [comment]
Who are you getting the update from? They probably don't have this fix.

**Comment by vvisigoth** [comment]
~marzod ?

**Comment by vvisigoth** [comment]
I just changed OTA source to ~binzod on the off chance that ~marzod didn't have this fix and I got same error (huge ames want/have)

**Comment by joemfb** [comment]
I take it all back, something (else?) must be wrong with the migration:

```
> -read %z ~zod %kids da+now /
0vg.kniql.spcko.ms91o.v2qng.8ah7k.evj1b.sbcko.3deer.3f5eq.fk0pn
> -read %z ~marzod %kids da+now /
0vg.kniql.spcko.ms91o.v2qng.8ah7k.evj1b.sbcko.3deer.3f5eq.fk0pn
> -read %z ~binzod %kids da+now /
0vg.kniql.spcko.ms91o.v2qng.8ah7k.evj1b.sbcko.3deer.3f5eq.fk0pn
```

Can you look at that massive have/need printf? Your ames version (`%$SOME_NUMBER`) should be the very first thing in `-have`.

**Comment by yosoyubik** [comment]
I've tried updating directly from ames-states %8 and %9 (separately, booting from 1.17 and 1.18 pills) directly to master, trying to reproduce this, and it seems to be working or at least I don't get any crash/nest-fail...
